test/index.ts

```diff
@@ -187,6 +189,27 @@ describe('Mutation', () => {
       expect(loginInfo.user).excluding(['password', 'id']).to.deep.equal(newUser);
       expect(+loginInfo.user.id).to.deep.equal(newUser.id);
       expect(loginInfo).to.haveOwnProperty('token');
+
+      const jwtPayload = jwt.verify(loginInfo.token, process.env.JWT_PRIVATE_KEY!) as jwt.JwtPayload;
```

Comment:
Agora essa primeira verificação acaba ja estando contida nas outras, né? Daria pra tirar.

Reply:
> Agora essa primeira verificação acaba ja estando contida nas outras, né? Daria pra tirar.

Não sei se entendi... eu uso o `jwt.verify` para extrair o payload to token, e os testes que vêm depois dele usam o payload que foi extraído. Como eu faria esses outros testes sem usar o `jwt.verify` antes?